internal/common/function/function.go

```diff
@@ -95,6 +95,11 @@ func ValidateBackupSpec(ctx context.Context, clientInstance client.Client, oadpN
 			return fmt.Errorf("NonAdminBackup spec.backupSpec.includedNamespaces can not contain namespaces other than: %s", nonAdminBackup.Namespace)
 		}
 	}
+
+	if nonAdminBackup.Spec.BackupSpec.ExcludedNamespaces != nil {
```

Comment:
This also means that the admins cannot enforce this spec value.

Reply:
what you mean by that?

we need code to check `enforcedBackupSpec` to do not allow admins enforcing it

Reply:
operator PR https://github.com/openshift/oadp-operator/pull/1632

Reply:
do we want to clarify which error can be lifted by admin to nonadmin users?